plugins/hls-semantic-tokens-plugin/test/testdata/T1.expected

```diff
@@ -72,7 +73,8 @@
 41:1-3 TFunction "go"
 41:6-9 TRecordField "foo"
 42:1-4 TFunction "add"
-42:7-18 TClassMethod "(Prelude.+)"
+42:8-16 TModuleName "Prelude."
```

Comment:
Can we have a test with a multiple-segment module name too? i.e. `(Foo.Bar.+)` or something

Reply:
improve in TQualifiedName.expected